tools/lou_checkyaml.c

```diff
@@ -484,12 +572,30 @@ read_test(yaml_parser_t *parser, char **tables, int direction, int hyphenation)
 
 	char **table = tables;
 	while (*table) {
-		if (cursorPos) {
-			if (xfail != check_cursor_pos(*table, word, cursorPos)) {
-				if (description) fprintf(stderr, "%s\n", description);
-				error_at_line(0, 0, file_name, event.start_mark.line + 1,
+		if (inPos || outPos || cursorPos) {
```

Comment:
Why is there this if a or b or c, followed by if a? Couldn't this just be a series if if a else if b else if c?

Reply:
Bue just adapted to the existing `else if (hyphenation)` below. I think there should be an error message though when cursorPos is specified in hyphenation test mode.